pypeit/core/skysub.py

```diff
@@ -433,16 +433,16 @@ def optimal_bkpts(bkpts_optimal, bsp_min, piximg, sampmask, samp_frac=0.80,
                 fillstyle='full', linestyle='None', label='optimal bkpts')
 
         ax.set_ylim((0.99 * sky_med_filt.min(), 1.01 * sky_med_filt.max()))
-        if mincol is not None:
-            plt.title(title_str + 'bkpt sampling spat pixels {:7.1f}-{:7.1f}'.format(mincol, maxcol))
+        if min_spat is not None:
+            plt.title(title_str + 'bkpt sampling spat pixels {:7.1f}-{:7.1f}'.format(min_spat, max_spat))
         plt.legend()
         plt.show()
 
     return fullbkpt
 
 
 def local_skysub_extract(sciimg, sciivar, tilts, waveimg, global_sky, rn2_img, thismask, slit_left, slit_righ, sobjs,
-                         bsp = 0.6, inmask = None, extract_maskwidth = 4.0, trim_edg = (3,3), std = False, prof_nsigma = None,
+                         spat_pix = None, bsp = 0.6, inmask = None, extract_maskwidth = 4.0, trim_edg = (3,3), std = False, prof_nsigma = None,
```

Comment:
Here and everywhere,  no space between variable and value.

`spat_pix=None`

Feel free to fix the rest of this call.

Reply:
Fixed.